library/windows/scripts/build_jsonlib.bat

```diff
@@ -12,94 +12,5 @@
-
-pushd %InstallDir%\VC\Auxiliary\Build
-call vcvarsall.bat x86_amd64
-popd
-
-set JSONDEBUGLIBEXISTS=true
-if not exist %~dp0..\dependencies\json\x64\debug\json_vc71_libmtd.lib set JSONDEBUGLIBEXISTS=false
-
-set JSONRELEASELIBEXISTS=true
-if not exist %~dp0..\dependencies\json\x64\release\json_vc71_libmt.lib set JSONRELEASELIBEXISTS=false
-
-if %JSONDEBUGLIBEXISTS% == true (
-    if %JSONRELEASELIBEXISTS% == true (
-        echo jsoncpp found.
-        goto DONE
-    )
-)
-
-set THIRDPARTYDIREXISTS=true
-if not exist %~dp0..\third_party set THIRDPARTYDIREXISTS=false
-
-if %THIRDPARTYDIREXISTS% == false (
-    mkdir %~dp0..\third_party
-)
-
-set JSONDIREXISTS=true
-if not exist %~dp0..\third_party\jsoncpp set JSONDIREXISTS=false
-
-if %JSONDIREXISTS% == false (
-    mkdir %~dp0..\third_party\jsoncpp
-)
-
-set JSONEXISTS=true
-if not exist %~dp0..\third_party\jsoncpp\README.md set JSONEXISTS=false
-
-:: Clone source
-if %JSONEXISTS% == false (
-    :: PR opened on json cpp for VS2017 support: https://github.com/open-source-parsers/jsoncpp/pull/853
-    echo Cloning via git clone --branch supportvs2017 https://github.com/clarkezone/jsoncpp.git %~dp0..\third_party\jsoncpp
-    call git clone --branch supportvs2017 https://github.com/clarkezone/jsoncpp.git %~dp0..\third_party\jsoncpp
-
-    pushd %~dp0..\third_party\jsoncpp
-
-    call git checkout 3ae7e8073a425c93329c8577a3c813c206322ca4
-
-    popd
-)
-
-:: Build debug lib
-echo Building debug lib: msbuild %~dp0..\third_party\jsoncpp\makefiles\msvc2017\lib_json.vcxproj 
-msbuild %~dp0..\third_party\jsoncpp\makefiles\msvc2017\lib_json.vcxproj
-
-set DEPBINDIREXISTS=true
-if not exist %~dp0..\dependencies\json\x64 set DEPBINDIREXISTS=false
-
-if %DEPBINDIREXISTS% == false (
-    mkdir %~dp0..\dependencies\json\x64
-)
-
-set DEPBINDBGDIREXISTS=true
-if not exist %~dp0..\dependencies\json\x64\debug set DEPBINDBGDIREXISTS=false
-
-if %DEPBINDBGDIREXISTS% == false (
-    mkdir %~dp0..\dependencies\json\x64\debug
-)
-
-copy %~dp0..\third_party\jsoncpp\makefiles\msvc2017\x64\debug\json_vc71_libmtd.lib %~dp0..\dependencies\json\x64\debug\.
-
-:: Build release lib
-echo Building release lib: msbuild %~dp0..\third_party\jsoncpp\makefiles\msvc2017\lib_json.vcxproj /p:Configuration=Release
-msbuild %~dp0..\third_party\jsoncpp\makefiles\msvc2017\lib_json.vcxproj /p:Configuration=Release
-
-set DEPBINRELDIREXISTS=true
-if not exist %~dp0..\dependencies\json\x64\release set DEPBINRELDIREXISTS=false
-
-if %DEPBINRELDIREXISTS% == false (
-    mkdir %~dp0..\dependencies\json\x64\release
-)
-
-copy %~dp0..\third_party\jsoncpp\makefiles\msvc2017\x64\release\json_vc71_libmt.lib %~dp0..\dependencies\json\x64\release\.
-
-:DONE
-echo jsoncpplib complete.
\ No newline at end of file
+for /f "delims=" %%i in ('%~dp0..\..\..\tools\flutter_location') do set FLUTTER_DIR=%%i
+%FLUTTER_DIR%\bin\cache\dart-sdk\bin\dart %~dp0..\..\..\tools\dart_tools\bin\build_jsoncpp.dart %~dp0..\third_party\jsoncpp %~dp0..\dependencies\JSON %*
```

Comment:
Let me know if you hit any issues switching these to run_dart_tool.bat, since it'll be the first real-world test of the script.